rust-runtime/aws-smithy-http-server/src/plugin/identity.rs

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::operation::Operation;
+
+use super::Plugin;
+
+/// An [`Plugin`] that maps an `input` [`Operation`] to itself.
+pub struct IdentityPlugin;
```

Comment:
Not related to this PR, but throwing it out as a discussion point: would `NoOpPlugin` be a better name for this?
I intuitively get what `identity` stands for here (math training, yay) but it feels like `no-op` would be clearer to a larger audience.

Reply:
I disagree, `NoOp` could be understood to mean that the plugin turns the operation into a no-op.

Reply:
I'm tempted to hide behind some existing precedents and ignore the confusion of the masses.

https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/struct.Identity.html
https://docs.rs/tower/0.4.8/tower/layer/util/struct.Identity.html
https://doc.rust-lang.org/std/convert/fn.identity.html

Reply:
I think `std` is a good enough precedent here 👌🏻